### PR TITLE
Create VCMI manifest

### PR DESCRIPTION
**Pull request by Alexander-Wilms**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. @ArseniyShestakov I couldn't register a forum account.  Would someone on the VCMI team be interested in (co-)maintaining this package?
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

This manifest doesn't use a specific VCMI tag or commit, since

_As of January 2021, we dont make big official releases anymore, but instead we provide a new version every several days: everytime when there is a new feature finished or new bug fixed.
Although there are no official "big releases", we reccomend updating the game at least twice a year, to make sure you have an updated, best version.
We try not to break savegame files compatibility, so you shouldnt have problem returning to your old games after update.
Please see corresponding "installation on" articles for details for your platform._

- https://wiki.vcmi.eu/Main_Page

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: eu.vcmi.VCMI.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          - sed -i 's|/usr/local|/app|g' Makefile
+        
+  - name: vcmi
+    buildsystem: cmake-ninja
+    config-opts:
+        - -Wno-dev
+    sources:
+      - type: git
+        url: https://github.com/vcmi/vcmi
+        branch: develop
```
Comment (hfiguiere) [inline eu.vcmi.VCMI.yaml:71]: you need a commit and no branch here.

**Comment by Alexander-Wilms** [comment]
bot, build eu.vcmi.VCMI

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: eu.vcmi.VCMI.yaml (hunk 1 above)
Comment (hfiguiere) [reply eu.vcmi.VCMI.yaml:71]: yes. otherwise the build isn't reproducible.

**Comment by Alexander-Wilms** [comment]
bot, build eu.vcmi.VCMI

**Comment by hfiguiere** [comment]
Why disabling aarch64? There is a version of luajit that builds on aarch64.
https://github.com/flathub/io.github.GnomeMpv/blob/ac259f54667bd82df9b13c67da1fb15544f41137/org.luajit.Luajit.json

**Comment by Alexander-Wilms** [comment]
Ah, okay. I didn't use 2.1.0 since it's still a beta.

**Comment by hfiguiere** [comment]
you could choose to use the beta ony for aarch64.

**Comment by Alexander-Wilms** [comment]
Just using one version is easier, if that's alright.

**Comment by Alexander-Wilms** [comment]
@hfiguiere Could you merge this?

**Comment by hfiguiere** [comment]
I can't, I don't have permissions.

**Pull request closed without merging**